sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 12.0.0-beta.12 (Unreleased)
+## 12.0.0 (2021-06-08)
```

Comment:
@Jinming-Hu can you please review all the changes since the last release and make sure the change log is complete and captures all relevant bug fixes and breaking changes?

Reply:
You can look at the commits and code changes from this range:
https://github.com/Azure/azure-sdk-for-cpp/compare/d99fcbc9f864ac63c57eabf780623dfc3571f7c5...17b91ffe598cf17bb4755480b70a9bb6fb3a02f3